view/theme/frio/templates/wall_thread.tpl

```diff
@@ -560,68 +560,95 @@ as the value of $top_child_total (this is done at the end of this file)
-								<a id="unpin-{{$item.id}}" href="javascript:doPin({{$item.id}});" class="btn-link {{$item.pin.classundo}}" title="{{$item.pin.undo}}"><i class="fa fa-dot-circle-o" aria-hidden="true"></i>&nbsp;{{$item.pin.undo}}</a>
-							</li>
-							{{/if}}
-
-							{{if $item.star}}
-								<li role="menuitem">
-								<a id="star-{{$item.id}}" href="javascript:doStar({{$item.id}});" class="btn-link {{$item.star.classdo}}" title="{{$item.star.do}}"><i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;{{$item.star.do}}</a>
-								<a id="unstar-{{$item.id}}" href="javascript:doStar({{$item.id}});" class="btn-link {{$item.star.classundo}}" title="{{$item.star.undo}}"><i class="fa fa-star" aria-hidden="true"></i>&nbsp;{{$item.star.undo}}</a>
-							</li>
-							{{/if}}
-
-							{{if $item.follow_thread}}
-								<li role="menuitem">
-								<a id="follow_thread-{{$item.id}}" href="javascript:{{$item.follow_thread.action}}" class="btn-link" title="{{$item.follow_thread.title}}"><i class="fa fa-plus" aria-hidden="true"></i>&nbsp;{{$item.follow_thread.title}}</a>
-							</li>
-							{{/if}}
-
-							{{if $item.language}}
-							<li role="menuitem">
-								<a id="language-{{$item.id}}" href="javascript:alert('{{$item.language.1}}');" class="btn-link filer-item language-icon" title="{{$item.language.0}}"><i class="fa fa-language" aria-hidden="true"></i>&nbsp;{{$item.language.0}}</a>
-							</li>
-							{{/if}}
-
-							{{if $item.ignore || ($item.drop && $item.drop.dropping)}}
-								<li role="separator" class="divider"></li>
-							{{/if}}
-
-							{{if $item.ignore}}
-								<li role="menuitem">
-								<a id="ignore-{{$item.id}}" href="javascript:doIgnoreThread({{$item.id}});" class="btn-link {{$item.ignore.classdo}}" title="{{$item.ignore.do}}"><i class="fa fa-bell-slash" aria-hidden="true"></i> {{$item.ignore.do}}</a>
-							</li>
-								<li role="menuitem">
-								<a id="unignore-{{$item.id}}" href="javascript:doIgnoreThread({{$item.id}});" class="btn-link {{$item.ignore.classundo}}"  title="{{$item.ignore.undo}}"><i class="fa fa-bell" aria-hidden="true"></i> {{$item.ignore.undo}}</a>
-							</li>
-							{{/if}}
-
-							{{if $item.drop && $item.drop.dropping}}
-								<li role="menuitem">
-								<a class="btn-link navicon delete" href="javascript:dropItem('item/drop/{{$item.id}}', 'item-{{$item.guid}}');" title="{{$item.drop.label}}"><i class="fa fa-trash" aria-hidden="true"></i> {{$item.drop.label}}</a>
-							</li>
-							{{/if}}
+										{{if $item.edpost}} {{* edit the posting *}}
+                                        <li role="menuitem">
+                                                <a href="javascript:editpost('{{$item.edpost.0}}?mode=none');" title="{{$item.edpost.1}}" class="btn-link navicon pencil"><i class="fa fa-pencil" aria-hidden="true"></i> {{$item.edpost.1}}</a>
```

Comment:
Once again the indentation has been switched from tabulations to spaces. Please check your text editor settings, it might do it automatically.

Here's the reference indentation for the Smarty condition, the `<li>` and the `<a>` tags, it should be applied to the whole change set.
```suggestion
						{{if $item.edpost}} {{* edit the posting *}}
							<li role="menuitem">
								<a href="javascript:editpost('{{$item.edpost.0}}?mode=none');" title="{{$item.edpost.1}}" class="btn-link navicon pencil"><i class="fa fa-pencil" aria-hidden="true"></i> {{$item.edpost.1}}</a>
```

Reply:
May i ask which Editor you are using?

Reply:
I'm using JetBrains PhpStorm with a paid subscription at the moment.